pkg/provider/cloud/kubevirt/namespace.go

```diff
@@ -67,3 +67,13 @@ func deleteNamespace(ctx context.Context, name string, client ctrlruntimeclient.
 
 	return client.Delete(ctx, ns)
 }
+
+func reconcileKubeVirtDiskCacheNamespace(ctx context.Context, name string, client ctrlruntimeclient.Client) error {
```

Comment:
We could group `reconcileKubeVirtDiskCacheNamespace` and `reconcileNamespace` in a `reconcileNamespaces` and reconcile both in the same ?